doc/asciidoc/pagerank.adoc

```diff
@@ -118,97 +120,100 @@ As we might expect, the Home page has the highest PageRank because it has incomi
-If `Person A` is a `FRIEND` with `Person B` we can say that `Person B` is also a `FRIEND` with `Person A`.
+Relationships in Neo4j always have a direction, but in this domain the direction is irrelevant.
+If `Person A` is a `FRIEND` with `Person B`, we can say that `Person B` is also a `FRIEND` with `Person A`.
 
-The default label and relationship-type selection syntax won't work for us here because it will project a directed social network.
+The default label and relationship-type selection syntax won't work for us here, because it will project a directed social network.
 Instead, we can project our undirected social network using *Cypher loading*.
 We can also apply this approach to other algorithms that use *Cypher loading*.
 
-.Running algorithm on Yelp social network
+Use the following to run the PageRank algorithm on the Yelp social network:
+
 [source,cypher]
 ----
 include::scripts/pagerank.cypher[tag=pagerank-stream-yelp-social]
 ----
 
 == Syntax
 
-.running algorithm and writing back results
+Use the following to run the PageRank algorithm, and write back results:
+
 [source,cypher]
 ----
 CALL algo.pageRank(label:String, relationship:String,
     {iterations:20, dampingFactor:0.85, write: true, writeProperty:'pagerank', concurrency:4})
 YIELD nodes, iterations, loadMillis, computeMillis, writeMillis, dampingFactor, write, writeProperty
 ----
 
-.parameters
+.Parameters
 [opts="header",cols="1,1,1,1,4"]
 |===
-| name | type | default | optional | description
-| label  | string | null | yes | label to load from the graph. If null load all nodes
-| relationship | string | null | yes | relationship-type to load from the graph. If null load all relationships
-| iterations | int | 20 | yes | how many iterations of page-rank to run
-| concurrency | int | available CPUs | yes | number of concurrent threads
-| dampingFactor | float | 0.85 | yes | damping factor of the page-rank calculation
-| write | boolean | true | yes | if result should be written back as node property
-| writeProperty | string | 'pagerank' | yes | property name written back to
-| graph | string | 'heavy' | yes | use 'heavy' when describing the subset of the graph with label and relationship-type parameter, 'cypher' for describing the subset with cypher node-statement and relationship-statement
+| Name | Type | Default | Optional | Description
+| label  | string | null | yes | The label to load from the graph. If null, load all nodes
+| relationship | string | null | yes | The relationship-type to load from the graph. If null, load all relationships
+| iterations | int | 20 | yes | How many iterations of PageRank to run
+| concurrency | int | available CPUs | yes | The number of concurrent threads
+| dampingFactor | float | 0.85 | yes | The damping factor of the PageRank calculation
+| write | boolean | true | yes | Specify if the result should be written back as a node property
+| writeProperty | string | `pagerank` | yes | The property name written back to
```

Comment:
not sure if the people then still know that they need to pass in a string value? if you remove the quotes, otherwise use 
```
`'pagerank'`
```